resources/views/index.blade.php

```diff
@@ -95,7 +95,9 @@
       SwaggerUIBundle.plugins.DownloadUrl
     ],
 
-    layout: "StandaloneLayout"
+    layout: "StandaloneLayout",
+
+    persistAuthorization: "{{config('l5-swagger.defaults.persist_authorization')}}",
```

Comment:
I think this should be `bool` (https://github.com/springdoc/springdoc-openapi/commit/8233e80dd41cda813864e3b052f4fa4906639c61#diff-8772e1e8e91643df2d1925f34074520faf154d857f786733e67c114ccfff53bcR172): 

```suggestion
    persistAuthorization: {!! config('l5-swagger.defaults.persist_authorization') ? 'true', 'false' !!},
```